kfs-purap/src/main/java/org/kuali/kfs/module/purap/PurapConstants.java

```diff
@@ -41,6 +32,15 @@
 import org.kuali.kfs.sys.KFSPropertyConstants;
 import org.kuali.rice.core.api.util.type.KualiDecimal;
 
+import java.util.ArrayList;

```

Comment:
It's an algorithm in IntelliJ for organizing imports. I would turn it off, but then we lose the "delete unused" functionality, and also organizing for when imports really are a mess. In the end, it doesn't matter to the compiler, but nonetheless frustrating for viewing by humans.